src/utils/Sorter.js

```diff
@@ -982,6 +982,35 @@ export default Backbone.View.extend({
     if (h) plh.style.height = h;
   },
 
+  /**
+   * Build an array of all the parents, including the component itself
+   * @return {Model|null}
+   */
+  parents(model) {
+    return model ? [model].concat(this.parents(model.parent())) : [];
+  },
+
+  /**
+   * Sort according to the position in the dom
+   * @param {Object} obj1 contains {model, parents}
+   * @param {Object} obj2 contains {model, parents}
+   */
+  sort(obj1, obj2) {
+    // common ancesters
+    const ancesters = obj1.parents.filter(p => obj2.parents.includes(p));
+    const ancester = ancesters[0];
+    if (!ancester)
+      throw new Error(
+        `Could not find a common ancester for the dragged elements #${obj1.model.getId()} and #${obj2.model.getId()}`
+      );
+    // find siblings in the common ancester
+    // the sibling is the element inside the ancester
+    const s1 = obj1.parents[obj1.parents.indexOf(ancester) - 1];
+    const s2 = obj2.parents[obj2.parents.indexOf(ancester) - 1];
+    // order according to the position in the DOM
+    return s2.index() - s1.index();
+  },
```

Comment:
Nice ❤️
I'm only wondering if `if (!ancester)` is possible and if we can at least fallback to a simple `obj2.model.index() - obj1.model.index()` instead of throwing error